ui/components/ui/icon/icon-block-explorer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const IconBlockExplorer = ({ className }) => (
+  <svg
+    className={className}
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path d="M8.69449 7.30553L14.3889 1.61108L8.69449 7.30553Z" fill="white" />
+    <path d="M14.9446 4.389V1.05566H11.6113" fill="white" />
+    <path d="M7.3056 1.05566H5.91671C2.44449 1.05566 1.0556 2.44455 1.0556 5.91678V10.0834C1.0556 13.5557 2.44449 14.9446 5.91671 14.9446H10.0834C13.5556 14.9446 14.9445 13.5557 14.9445 10.0834V8.69455" />
+    <path
+      d="M8.69449 7.30553L14.3889 1.61108M14.9446 4.389V1.05566H11.6113M7.3056 1.05566H5.91671C2.44449 1.05566 1.0556 2.44455 1.0556 5.91678V10.0834C1.0556 13.5557 2.44449 14.9446 5.91671 14.9446H10.0834C13.5556 14.9446 14.9445 13.5557 14.9445 10.0834V8.69455"
+      stroke="#BBC0C5"
+      strokeWidth="1.5"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
```

Comment:
Can optimize svg and use new icon format here

```suggestion
const IconBlockExplorer = ({
  size = 24,
  color = 'currentColor',
  ariaLabel,
  className,
  onClick,
}) => (
  <svg
    width={size}
    height={size}
    fill={color}
    className={className}
    aria-label={ariaLabel}
    onClick={onClick}
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 512 512"
  >
    <path d="m328 482l-128 0c-115 0-168-52-168-169l0-128c0-115 53-168 168-168l43 0c5 0 10 2 14 6 3 3 5 8 5 13 0 5-2 10-5 14-4 3-9 5-14 5l-43 0c-96 0-130 34-130 130l0 128c0 96 34 131 130 131l128 0c96 0 130-35 130-131l0-42c0-5 3-10 6-14 4-3 9-5 14-5 5 0 10 2 13 5 4 4 6 9 6 14l0 42c0 117-52 169-169 169z m-42-235c-5 0-10-2-14-5-3-4-5-9-5-14 0-5 2-10 5-13l159-160-56 0c-5 0-10-2-13-5-4-4-6-9-6-14 0-5 2-10 6-13 3-4 8-6 13-6l103 0c2 0 5 0 7 1 2 1 5 3 6 5 2 1 4 3 5 6 0 2 1 5 1 7l0 103c0 5-2 10-6 13-3 4-8 6-13 6-5 0-10-2-14-6-3-3-6-8-6-13l0-56-159 159c-3 3-8 5-13 5z" />
  </svg>
);
```

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).